app/views/samples/_filters.haml

```diff
@@ -25,3 +25,10 @@
             = cdx_select name: "specimen_role", value: params["specimen_role"] do |specimen_role|
               - specimen_role.item "", "All specimen roles"
               - specimen_role.items Sample.specimen_roles.map{|item| {id: item[:id], description: item[:description].truncate(30) }}, :id, :description
+          .filter-group
+            .filter
+              %label.block Modified From
+              %input.input-large{type: "date", name: "updated_at_from", value: params["updated_at_from"]}
+            .filter
+              %label.block Modified To
+              %input.input-large{type: "date", name: "updated_at_to", value: params["updated_at_to"]}
```

Comment:
If you keep the `filled` class, you could specify it right into the HTML, so you wouldn't need to run a JS function on page load to set it. For example:

```suggestion
              %input.input-large{type: "date", name: "updated_at_to", value: params["updated_at_to"], class: ("filled" if params["updated_at_to"].present?)}
```

Reply:
The JS function is not just for content that loads from server. But when the users clears the field, it will work too. That's why I did it through javascript, and I realized too that this is a change I had to do on every input date. So it works for all.